README.md

```diff
@@ -142,12 +137,26 @@ The following methods are exposed on an instantiated client
 | addReactors | `(() => {customReactors}) => null` | *Experimental* A method for adding new reactors on a per instance basis. You pass in a function which returns an object of new reactors to support (`{reactorName: reactorFunction}`). Reactors are passed `(parent, node, event, reactor, types, lookup)` and are expected to return an array of affected nodes. |
 | subquery | `(targetPath, sourceTree, sourcePath, mapSubqueryValues?) => {}` | Sets up a subquery, using the types passed in to the client and assuming this tree instance is the target tree. For more info, see the [subquery](#Subquery) section below. |
-| replace | `async (path, node) -> await searchCompleted` | Replaces a the node at the given path with the new node. You can await this for when updates settle and relevant searches are completed. |
-| wrapInGroup | `async (path, newNode) -> await searchCompleted` | Wraps the node at the provided path in a group described by `newNode`. The node will be replaced it on its parent's children unless it is root node (with a nul parent). If the node has no parent, it will be done in place by mutating the node into the group described by `newNode` with the original node as it's only child. You can await this for when updates settle and relevant searches are completed. |
-| move | `async (path, { path, index }) -> await searchCompleted` | Moves the node at the provided path (first argument) to the target location provided in the second parameter. If the target path is not specified, it will default to the current node's group. If the target index isn't provided, it will default to moving to the end of the target group. You can await this for when updates settle and relevant searches are completed. Useful for drag and drop query builder interfaces that let you move nodes around in and between groups. |
-| pauseNested | `async (path) -> await searchCompleted` | Recursively set paused to true for the node at the path and all its children. You can await this for when updates settle and relevant searches are completed. |
-| unpauseNested | `async (path) -> await searchCompleted` | Recursively set paused to false for the node at the path and all its children. You can await this for when updates settle and relevant searches are completed. |
-| isPausedNested | `async (path) -> bool` | Returns a bool for whether the node at the path and all of its children are paused. |
+
+#### Actions
+
+Client methods that mutate or otherwise act on nodes are known as **actions**. Except for `isPausedNested`, you can await these for when updates settle and relevant searches are completed. Conventionally, actions take a node path as their first parameter. 
```

Comment:
Took the opportunity to factor out the "you can await this" bit that was in the descriptions for a lot of these - I think it should be fine like this, but lemme know if it's not clear enough